client/blocks/reader-featured-video/index.jsx

```diff
@@ -50,7 +51,7 @@ class ReaderFeaturedVideo extends React.Component {
 	};
 
 	setVideoSizingStrategy = ( videoEmbed ) => {
-		let sizingFunction = constant( {} );
+		let sizingFunction = defaultSizingFunction;
```

Comment:
`constant( {} )` returns an empty object, while `() => {}` returns `undefined`. Does it make a difference in this case?

Reply:
I think this was overlooked by me last time. Fixed